pynbody/sph/kd.h

```diff
@@ -1,7 +1,7 @@
 #ifndef KD_HINCLUDED
 #define KD_HINCLUDED
 
-#include <Python.h>
+#include <python3.8/Python.h>
```

Comment:
This is causing builds to fail because it assumes Python 3.8

Reply:
Thanks, fixed it.